packages/artifact/src/internal/shared/artifact-twirp-client.ts

```diff
@@ -64,42 +64,57 @@ class ArtifactHttpClient implements Rpc {
         this.httpClient.post(url, JSON.stringify(data), headers)
       )
 
-      return JSON.parse(body)
+      return body
     } catch (error) {
       throw new Error(`Failed to ${method}: ${error.message}`)
     }
   }
 
   async retryableRequest(
     operation: () => Promise<HttpClientResponse>
-  ): Promise<{response: HttpClientResponse; body: string}> {
+  ): Promise<{response: HttpClientResponse; body: object}> {
```

Comment:
Just made this return an object since we're always going to be JSON parsing the response